src/index.js

```diff
@@ -0,0 +1,20 @@
+import { renderPage } from './gui';
+import { getState } from './state';
+
+// Initialize projects storage
+//
+window.projects = [];
+
+// Check local storage for prior usage
+// if (localStorage.getItem('projects')) {
+//   window.projects = expandState(localStorage.getItem('projects'));
+// } 
+
+const root = document.querySelector('main');
+
+getState().then(() => {
+  renderPage(root, window.projects[0], window.projects);
```

Comment:
Do not use Window.alert() or Window.confirm(). It’s ugly, impossible to style, stop code execution and display differently on different browsers. Use custom modal instead


The previous reviewer @rindrajosia meant you should remove `window.`